libc/test/src/sys/ioctl/linux/ioctl_test.cpp

```diff
@@ -0,0 +1,29 @@
+//===-- Unittests for ioctl ---------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
These two longer comment lines don't appear to be the same length. Does running the formatter fix this?

Reply:
I copied from the other test file and change the syscall name, running the formatter does not seem to fix that, so I fixed that manually.